### PR TITLE
Merge discvr-20.11 to develop

### PR DESCRIPTION
**Pull request by bbimber**

### CODE REVIEW

File: mcc/resources/etls/wnprc.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
     <!--species~eq=Marmoset-->
     <incrementalFilter className="ModifiedSinceFilterStrategy" timestampColumnName="modified" />
     <schedule>
-        <!--11PM-->
-        <cron expression="0 0 23 * * ?"/>
+        <!--101PM-->
```
Comment (labkey-jeckels) [inline mcc/resources/etls/wnprc.xml:16]: This is 8PM, right?

File: mcc/src/org/labkey/mcc/MccModule.java [hunk 2]
```diff
@@ -81,4 +82,23 @@ public void doStartup(ModuleContext moduleContext)
...
+    {
+        EHRService.get().registerModule(this);
+        //EHRService.get().registerTableCustomizer(this, ONPRC_EHRCustomizer.class);
+
+        //Resource r = getModuleResource("/scripts/mcc/mcc_triggers.js");
+        //assert r != null;
+        //EHRService.get().registerTriggerScript(this, r);
+
+        //EHRService.get().registerClientDependency(ClientDependency.supplierFromPath("Ext4"), this);
+        //EHRService.get().registerClientDependency(ClientDependency.supplierFromPath("onprc_ehr/panel/BloodSummaryPanel.js"), this);
```
Comment (labkey-jeckels) [inline mcc/src/org/labkey/mcc/MccModule.java:96]: Is it safe to delete these commented lines?

**Review by labkey-jeckels — APPROVED** [review]
Please hold the merge until the TeamCity configs are happy on develop, but approving to help move things forward when it's ready.

A couple of small comments, up to you if it's worth addressing.

File: mcc/src/org/labkey/mcc/MccModule.java (hunk 2 above)
Comment (bbimber) [reply mcc/src/org/labkey/mcc/MccModule.java:96]: yeah, this module is very much in-progress. some of it was a placeholder for me to remember what i need to do

File: mcc/resources/etls/wnprc.xml (hunk 1 above)
Comment (bbimber) [reply mcc/resources/etls/wnprc.xml:16]: correct, not 101PM

**Pull request merged**